vmdb/db/migrate/20150111161446_create_nodes.rb

```diff
@@ -3,7 +3,7 @@ def up
     create_table :container_nodes do |t|
       t.string    :ems_ref
       t.string    :name
-      t.timestamp  :creation_timestamp
+      t.timestamp  :creation_timestamp, :null => true
```

Comment:
Is the default for any timestamp column to be not-null?  or is it only for the timestamps method.  If so, why was this changed?  Seems so arbitrary. @tenderlove ?